src/parsley/parsley_message.py

```
@@ -0,0 +1,49 @@
+    """
+
+    board_type_id: BoardTypeID
+    board_inst_id: BoardInstID
+    msg_prio: MsgPrio
+    msg_type: MsgType
+    data: T # ParsleyDataType
+
+    @field_validator("msg_prio")
+    def validate_msg_prio(cls, value):
+        if value not in mt.msg_prio:
+            raise ValueError(f"Invalid msg_prio type '{value}'")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)